groups/models.py

```diff
@@ -82,5 +82,12 @@ class Comment(models.Model):
     class Meta:
         ordering = ('date_created',)
 
+    def get_pagejump_anchor(self):
+        return 'c{}'.format(self.pk)
```

Comment:
It trims down the URLs, which is where this is used.  It's not meant to be user-friendly, I just wanted something unobtrusive.  (I started with `comment` and shortened it later when I realised the full word was unnecessary.)